main.tf

```diff
@@ -2,7 +2,7 @@
 data "aws_organizations_organization" "organization" {}
 
 locals {
-  json_template           = file(".terraform/modules/drata_role_cloudformation_stacksets/drata_cloudformation_stackset_template.json")
+  json_template           = file("${path.module}/drata_cloudformation_stackset_template.json")
```

Comment:
This was suggested in [this](https://github.com/drata/aws-cloudformation-drata-setup/pull/4) PR and is required to run this module correctly